aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -2308,3 +2350,23 @@ resource "aws_spot_fleet_request" "test" {
 }
 `, rName, validUntil)
 }
+
+func testAccAWSSpotFleetRequestZeroCapacityConfig(rName string, validUntil string) string {
+	return testAccAWSSpotFleetRequestConfigBase(rName) + fmt.Sprintf(`
+resource "aws_spot_fleet_request" "test" {
+    iam_fleet_role = "${aws_iam_role.test.arn}"
+    spot_price = "0.005"
```

Comment:
`spot_price` seems to have reverted.

```suggestion
    spot_price = "0.05"
```